### PR TITLE
like implementation

### PR DESCRIPTION
**Pull request by MaziarHeidari2003**
I made some changes in the posts.models, posts,urls, posts.serializers and posts.urls . I wish that would be enough for backend to add the like functionality. I tried to use the existing classes in views but I couldn't do it! 
So I made a new CBW class called  LikePostAPIView. check this out.

### CODE REVIEW

File: server/posts/models.py
Comment (hosseincodes) [inline server/posts/models.py:1]: why did you add blank=true here? the number of likes should always be a numerical value

**Review by hosseincodes — CHANGES_REQUESTED** [review]
well done Maziar! we need posts liked by an account to be on their profile page as well, to implement this you have to made a liked-posts field in accounts.modle

File: server/posts/models.py
Comment (MaziarHeidari2003) [reply server/posts/models.py:1]: Hi there dear Hossein, you've got a point about the blank=True in the likes field. I was just thinking what value should it take when it is initially being created in a post by a user. But the thing is that I cannot really think of a default value for this filed ! Because this field is not about counting or numbers, this field is just a relationship between the post and other users. So I think it your concern about having a numerical value can be solved by the likes_count method . I no one has liked the post it will return 0. I think this is the most special case, isn't it?
